dspace/config/log4j2.xml

```diff
@@ -76,15 +76,40 @@
                 <policy type='TimeBasedTriggeringPolicy'>yyyy-MM-dd</policy>
             </policies>
         </Appender>
+
+        <!-- A4 is for Warns and errors -->
+        <Appender name='A4'
+                  filePattern="${log.dir}/warn.log-%d{yyyy-MM-dd}"
```

Comment:
It is not only warnings - maybe name it like `error-warn.log..`

Reply:
so debug would be called `debug-info-warn-wrror-fatal.log`? I don't agree with this idea.

Reply:
No, but I would not search errors in the `warn.log`. From the name it should be clear that there are warnings and errors.